cypress/support/commands.ts

```diff
@@ -33,23 +38,27 @@ export enum HoursInfoCells {
   overtime = 2,
 }
 export type ScheduleName = "example.xlsx" | "grafik.xlsx" | "example_2.xlsx";
+const numberOfDaysInWeek = 7;
```

Comment:
tam wyżej komentowałem, że loadScheduleToMonth się powinien nazywać loadScheduleFromFile, a teraz w sumie jakby zrobić obiekt ScheduleName później wywoływać loadScheduleToMonth(ScheduleName.january2020) to może by miało sens

Reply:
Nie do końca rozumiem. Chciałbyś stworzć obiekt ScheduleName, który będzie zawierał inroamcje o nazwie pliku i miesiącu, na który mamy wgrać harmonogram? Bo istotne tutaj jest to, że chcemy mieć taką funkcjonalność, że wgrywasz dowolny plik na dowolny miesiąc. Może być w pliku napisane Listopad 2020, ale ja mogę chcieć wgrać to na luty, tylko muszę go odpowiednio dopasować i głównie to miałem wprowadzić w tym tasku. Z tym, że wyszło jeszcze 100000 innych rzeczy XD

Reply:
Znaczy się spoko, jeśli to nie ma znaczenie co to za miesiąc to spoko, można poprostu nazwać funkcję loadScheduleFromFile